libs/libavro/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=avro-c
+PKG_VERSION:=1.8.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://www.apache.org/dist/avro/avro-$(PKG_VERSION)/c/
```

Comment:
PKG_SOURCE_URL:=@APACHE/avro/avro-$(PKG_VERSION)/c

Use the official mirror list and no trailing slash